shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/resources/sql/supported/ddl/create-table.xml

```diff
@@ -113,122 +102,15 @@
-    <sql-case id="create_index_using_gist" value="CREATE INDEX pointloc ON t_order USING gist (func1(location,location))" db-types="PostgreSQL" />
-    <sql-case id="create_index_with_concurrently" value="CREATE INDEX CONCURRENTLY sales_quantity_index ON t_order(quantity)" db-types="PostgreSQL" />
-    <sql-case id="create_index_with_double_quota" value="CREATE INDEX &quot;order_index&quot; ON &quot;t_order&quot; (&quot;order_id&quot;)" db-types="PostgreSQL" />
-    <sql-case id="create_index_with_double_quota_and_uescape" value="CREATE INDEX U&amp;&quot;order_index!0061t!+000061&quot; UESCAPE '!' ON u&amp;&quot;t_order&quot; (u&amp;&quot;order_id&quot;)" db-types="PostgreSQL" />
-    <sql-case id="create_index_concurrently" value="CREATE INDEX CONCURRENTLY order_index ON t_order (order_id)" db-types="PostgreSQL" />
-    <sql-case id="create_index_if_not_exists" value="CREATE INDEX IF NOT EXISTS order_index ON t_order (order_id)" db-types="PostgreSQL" />
-    <sql-case id="create_index_using_btree" value="CREATE INDEX order_index ON t_order (order_id) USING BTREE" db-types="PostgreSQL" />
-    <sql-case id="create_index_with_unique" value="CREATE UNIQUE INDEX order_index ON t_order (order_id)" db-types="Oracle" />
-    <sql-case id="create_index_with_usable" value="CREATE UNIQUE INDEX order_index ON t_order (order_id) USABLE" db-types="Oracle" />
-    <sql-case id="create_index_with_unusable" value="CREATE UNIQUE INDEX order_index ON t_order (order_id) UNUSABLE" db-types="Oracle" />
-    <sql-case id="create_index_with_immediate_invalidation" value="CREATE UNIQUE INDEX order_index ON t_order (order_id) IMMEDIATE INVALIDATION" db-types="Oracle" />
-    <sql-case id="create_index_with_cluster_index" value="CREATE UNIQUE INDEX order_index ON CLUSTER t_cluster" db-types="Oracle" />
-    <sql-case id="create_index_with_bitmap" value="CREATE BITMAP INDEX order_index ON t_order (order_id)" db-types="Oracle" />
-    
-<!--    create database test-->
-    <sql-case id="create_database" value="CREATE DATABASE lusiadas" db-types="PostgreSQL" />
-    <sql-case id="create_database_owner" value="CREATE DATABASE sales OWNER salesapp TABLESPACE salesspace" db-types="PostgreSQL" />
-    <sql-case id="create_database_with_location_and_encoding" value="CREATE DATABASE music2
-        LC_COLLATE 'sv_SE.iso885915' LC_CTYPE 'sv_SE.iso885915'
-        ENCODING LATIN9
-        TEMPLATE template0" db-types="PostgreSQL" />
-    <sql-case id="create_database_with_location" value="CREATE DATABASE music
-        LC_COLLATE 'sv_SE.utf8' LC_CTYPE 'sv_SE.utf8'
-        TEMPLATE template0" db-types="PostgreSQL" />
-
-<!--    create function test-->
-    <sql-case id="create_function" value="CREATE FUNCTION add(integer, integer) RETURNS integer
-        AS 'select $1 + $2;'
-        LANGUAGE SQL
-        IMMUTABLE
-        RETURNS NULL ON NULL INPUT" db-types="PostgreSQL" />
```

Comment:
So we just move these tests to other files?

Reply:
yes, e.g. I just divided `alter` into `alter-table`, `alter-database`... like what we have in `shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/resources/case/ddl` for a better structure:)